edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc

```diff
@@ -6,44 +6,53 @@ include::_attributes/common-attributes.adoc[]
+* xref:../../edge_computing/image_based_upgrade/cnf-understanding-image-based-upgrade.adoc#cnf-understanding-image-based-upgrade[Understanding the image-based upgrade for single-node OpenShift clusters]
 
-* xref:../../edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/ztp-image-based-upgrade-prep-resources.adoc#ztp-image-based-upgrade-prep-resources[Creating ConfigMap objects for the image-based upgrade with {lcao} using {ztp}]
+//include::modules/ztp-image-based-upgrade-concept.adoc[leveloffset=+1]
+
+include::modules/ztp-image-based-upgrade-procedure-steps.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
 
 * xref:../../edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-shared-container-partition.adoc#ztp-image-based-upgrade-shared-container-partition_shared-container-partition[Configuring a shared container partition between ostree stateroots when using {ztp}]
 
+* xref:../../edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/ztp-image-based-upgrade-prep-resources.adoc#ztp-image-based-upgrade-prep-resources[Creating ConfigMap objects for the image-based upgrade with {lcao} using {ztp}]
+
 * xref:../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc#oadp-about-backup-snapshot-locations_installing-oadp-ocs[About backup and snapshot locations and their secrets]
 
 * xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc#oadp-creating-backup-cr-doc[Creating a Backup CR]
 
 * xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-cr_restoring-applications[Creating a Restore CR]
 
-include::modules/ztp-image-based-upgrade-upgrade.adoc[leveloffset=+1]
+//* xref:../../edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc#ztp-image-based-upgrade-supported-combinations_ztp-gitops[Supported action combinations]
+
+include::modules/ztp-image-based-upgrade-procedure-one-step.adoc[leveloffset=+1]
+
+include::modules/ztp-image-based-upgrade-procedure-cancel.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../../edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc#ztp-image-based-upgrade-rollback_ztp-gitops[Moving to the Rollback stage of the image-based upgrade with {lcao} and {ztp}]
-
-* xref:../../edge_computing/cnf-talm-for-cluster-upgrades.adoc#talo-policies-concept_cnf-topology-aware-lifecycle-manager[Update policies on managed clusters]
+//* xref:../../edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc#ztp-image-based-upgrade-supported-combinations_ztp-gitops[Supported action combinations]
```

Comment:
An additional resources heading is in the wrong location: 

![Screenshot from 2025-01-09 17-02-56](https://github.com/user-attachments/assets/6d1b8d57-bb5b-440d-bfdf-344913cbbb35)

This needs to be fixed before I can mark the peer review as done.